src/scripts/listener.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-cycle */
+import popUpAnime from './popUp.js';
+import { newSession } from './calls.js';
+import showComments from './showComments.js';
+import refresher from './refresher.js';
+import saveLike from './saveLike.js';
+
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/S7zVMxWAOezBiRHSLLWC/comments';
+
+export default class eventsPopUp {
+  // Open PopUp window
+  static popUpEvent() {
+    const btns = document.querySelectorAll('button');
+    btns.forEach((btn) => {
+      btn.addEventListener('click', (e) => {
+        const container = document.querySelector('.popup');
+        container.style = 'height: 100vh; width: 100vw;';
```

Comment:
[Optional]
You should use CSS for styling elements. Inline styling makes maintaining your pages harder.